iothub/device/src/Authentication/IotHubConnectionCredentials.cs

```diff
@@ -210,94 +169,53 @@ private void ExtractPropertiesFromConnectionString(string iotHubConnectionString
             SharedAccessKeyName = GetConnectionStringOptionalValue(map, SharedAccessKeyNamePropertyName);
             SharedAccessKey = GetConnectionStringOptionalValue(map, SharedAccessKeyPropertyName);
             SharedAccessSignature = GetConnectionStringOptionalValue(map, SharedAccessSignaturePropertyName);
-            UsingX509Cert = GetConnectionStringOptionalValueOrDefault<bool>(map, X509CertPropertyName, ParseX509, true)
```

Comment:
this needs to be set correctly now that connection string no longer expects x509=true